### DISCUSSION TITLE
[Bug] Invoke-RestMethod use proxy of scoop

### DISCUSSION THREAD

**Discussion by zxw7606** [ScoopInstaller/Scoop#5673] [Help and Questions] [answered]
#### Current Behavior

I am trying to write a script like this and I configured a wrong proxy of scoop
```powershell
scoop -v
Invoke-RestMethod "https://raw.githubusercontent.com/scoopinstaller/install/master/install.ps1"
```

and the Invoke-RestMethod will force use proxy

```
+     Invoke-RestMethod "https://raw.githubusercontent.com/scoopinstall ...
+     ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : InvalidOperation: (System.Net.HttpWebRequest:HttpWebRequest) [Invoke-RestMethod]，
    WebException
    + FullyQualifiedErrorId : WebCmdletWebResponseException,Microsoft.PowerShell.Commands.InvokeRestMethodComma
   nd
```

#### Expected Behavior

Invoke-RestMethod not use scoop proxy
### System details

**Windows version:** [e.g. 7, 8, 10, 11]
10
**OS architecture:** [e.g. 32bit, 64bit, arm64]
64bit
**PowerShell version:** [output of `"$($PSVersionTable.PSVersion)"`]
5.1
**Additional software:** [(optional) e.g. ConEmu, Git]

#### Scoop Configuration

```json
{
    "proxy":  "127.0.0.1:10809",
    "scoop_branch":  "master",
    "last_update":  "2023-05-11T21:29:43.2148099+08:00",
    "scoop_repo":  "https://github.com/ScoopInstaller/Scoop"
}

```

**Comment by rashil2000** [comment]
Could you please explain in detail the use case?

**Reply by HUMORCE** [reply, marked as answer]
This is a bug, same as https://github.com/ScoopInstaller/Scoop/issues/5435

**Reply by rashil2000** [reply]
Oh, okay
